xl_auth/translations/sv/LC_MESSAGES/messages.po

```diff
@@ -755,14 +770,18 @@ msgstr "Bearer-tokens för OAuth2"
 msgid "Delete Token"
 msgstr "Ta bort token"
 
-#: xl_auth/templates/permissions/edit.html:8
+#: xl_auth/templates/permissions/edit.html:5
 msgid "Edit Existing Permission"
 msgstr "Ändra behörighet"
 
 #: xl_auth/templates/permissions/home.html:9
 msgid "Existing Permissions"
 msgstr "Befintliga behörigheter"
 
+#: xl_auth/templates/permissions/home.html:49
+msgid "Delete Permission"
+msgstr "🙀 Ta bort behörighet"
```

Comment:
No emojis...

Reply:
@mxtthias  Are you sure?

Reply:
How about having a limit of 2 emojis hidden in places where people are very-very unlikely to ever get offended by them? 😅